### ISSUE TITLE
Customer Dashboards: Entity alias > Group entities

### ISSUE THREAD

**Issue by Kevinowitsch** [thingsboard/thingsboard#8768]
When managing / creating dashboards for customers I'd like to use their device groups as entity alias in the Dashboards.
But trying to do this while logged in as tenant, you can only see your own device groups, not the customer's. the only way to get it to work is to log in as customer and edit the dashboard.
Is there a way to do it without having to log in as customer?
If not that would be a nice feature to add.

**Comment by devaskim** [comment]
Agree, ownership entity alias is necessary for management customer's entities being logged in as tenant administrator.

I solved similar case by creating relation between customer and his/her users.

**Comment by dwinkler-oebb** [comment]
We also think this is a valid request since we use "customers" to separate different "IoT Usecases" from each.
We are building the dashboards while being logged in as "Tenant Admin" inside the "customers" and like to create "dashboard aliases" for "customer device groups" etc but can not select those:
![image](https://github.com/thingsboard/thingsboard/assets/88032369/76da036c-31c5-4b66-92c8-06b14c09fee0)

Maybe in the modal a new "checkbox" for "include customer entites" would be fine: 
![image](https://github.com/thingsboard/thingsboard/assets/88032369/7ee10d80-3174-4017-bc3f-ff8625cd370c)

Or alternatively "show only the entities of the owner customer of the dashboard" by default, not the owner of the current user.

**Comment by sparvalarva** [comment]
Hi,
Logged in here to post a new issue but found this one instead. 

We have the same kind of problem. 
Im building dashboards and want to create an alias for a group. Logged in as tenant admin i cant see the groups of the customers only the groups of the Tenant. 

My workaround is to create a Customer Admin account, log in with that and create the alias.

**Comment by Mike2912** [comment]
+1

**Comment by GDXFR** [comment]
Any news on this? Thanks
